src/Http/Request.php

```diff
@@ -176,8 +177,34 @@ public function getCookies()
 		return $this->cookies;
 	}
 
-
-	/********************* method & headers ****************d*g**/
+    /**
+     * Provides a flexible generic get method.
+     * It follows an order of precedence: GET, POST, FILE.
+     *
+     * @param $key
+     * @param $default
+     *
+     * @return mixed
+     */
+    public function get($key, $default = NULL)
```

Comment:
`$_REQUEST` only includes cookies if `request_order` is misconfigured. It does not include them by default.